prov/efa/src/rdm/rxr_ep.c

```diff
@@ -794,15 +794,6 @@ static int rxr_ep_ctrl(struct fid *fid, int command, void *arg)
 
 		rxr_ep_set_extra_info(ep);
 
-		ep->core_addrlen = RXR_MAX_NAME_LENGTH;
```

Comment:
I think the commit to remove `RXR_MAX_NAME_LENGTH` can be squashed into this commit

Reply:
Rebase is a little tricky here because commits modified same file `rxr.h`. Do you have strong preference?

Reply:
No strong preference